### PR TITLE
Bump micrometer-bom from 1.7.0 to 1.7.2

### LINKED ISSUE #18151: Upgrade micrometer-bom from 1.7.0 to 1.7.1 (or further)
## Description

Follow up from #18120 

Apparently the micrometer jar now include native-image metadata, and the included directives aren't correct.

## Implementation ideas

We will either need to contribute fixes to Micrometer or find a way to exclude them.

### PR DESCRIPTION
**Pull request by ebullient**
Resolves #18151

### CODE REVIEW

**Review by geoand — APPROVED** [review]

**Comment by gsmet** [comment]
We need both `backport?` for 2.1 and `backport-2.0?` for 2.0.

**Comment by ebullient** [comment]
I added that and then wasn't sure we'd do both and took the 2.0 one off.. I was going to ask you. ;)

File: bom/application/pom.xml [hunk 1]
```diff
@@ -32,7 +32,7 @@
         <jaeger.version>1.4.0</jaeger.version>
         <quarkus-http.version>4.1.1</quarkus-http.version>
         <jboss-servlet-api_4.0_spec.version>1.0.0.Final</jboss-servlet-api_4.0_spec.version>
-        <micrometer.version>1.7.0</micrometer.version> <!-- when updating, verify if com.google.auth should not be updated too -->
```
Comment (gsmet) [inline bom/application/pom.xml:35]: I'm surprised you removed the comment? We do not need google-auth to be aligned anymore?

Comment (ebullient) [reply bom/application/pom.xml:35]: Nope. The google stuff was related to stackdriver, which moved out to quarkiverse

**Pull request merged**